src/test/libpmempool_bttdev/TEST11.PS1

```diff
@@ -8,7 +8,9 @@
 . ..\unittest\unittest.ps1
 
 require_test_type medium
-require_fs_type any
+# dax on windows doesn't support sparse files
+require_fs_type non-pmem
```

Comment:
`require_sparse_support` ?

Reply:
I removed this requirement, this test makes sense on Dax too

Reply:
I meant that instead of relying on current implementation of pmem on Windows, you should add a new requirement that explicitly checks what you want.